site/configure.xml

```diff
@@ -497,71 +702,87 @@ CONFIG_FILE=/etc/rabbitmq/testdir/bunnies</pre>
           </tr>
           <tr>
             <td>
-              <code>tcp_listeners</code>
+              <code>listeners</code>
             </td>
             <td>
-              List of ports on which to listen for AMQP connections
-              (without SSL). Can contain integers (meaning "listen on
-              all interfaces") or tuples such as <code>{"127.0.0.1",
-              5672}</code> to listen on one or more interfaces.
+              Ports or hostname/pair on which to listen for AMQP connections
+              (without TLS). See the <a href="/networking.html">Networking guide</a> for more
+              details and examples.
 
-              <p>Default: <code>[5672]</code></p>
+              <p>Default: <code>listeners.tcp.default = 5672</code></p>
             </td>
           </tr>
           <tr>
-            <td><code>num_tcp_acceptors</code></td>
+            <td><code>num_acceptors.tcp</code></td>
             <td>
               Number of Erlang processes that will accept connections for the TCP
               listeners.
-              <p>Default: <code>10</code></p>
+              <p>Default: <code>num_acceptors.tcp = 10</code></p>
             </td>
           </tr>
           <tr>
             <td><code>handshake_timeout</code></td>
             <td>
-              Maximum time for AMQP 0-8/0-9/0-9-1 handshake (after socket connection
-              and SSL handshake), in milliseconds.
-              <p>Default: <code>10000</code></p>
+              Maximum time for AMQP 0-9-1 handshake (after socket connection
+              and TLS handshake), in milliseconds.
+              <p>Default: <code>handshake_timeout = 10000</code></p>
             </td>
           </tr>
           <tr>
-            <td><code>ssl_listeners</code></td>
+            <td><code>listeners.ssl</code></td>
             <td>
-              As above, for SSL connections.
-              <p>Default: <code>[]</code></p>
+              Ports or hostname/pair on which to listen for AMQP connections
+              (without TLS). See the <a href="/networking.html">Networking guide</a> for more
+              details and examples.
+              <p>Default: <code>listeners.ssl = none</code></p>
             </td>
           </tr>
           <tr>
-            <td><code>num_ssl_acceptors</code></td>
+            <td><code>num_acceptors.ssl</code></td>
             <td>
-              Number of Erlang processes that will accept connections for the SSL
+              Number of Erlang processes that will accept connections for the TLS
```

Comment:
Yes but not in scope for this issue. I do it as drive-by changes whenever I work on the docs.